### ISSUE TITLE
Hardware register default value doesn't make sense on DMG for CGB-only registers

### ISSUE THREAD

**Issue by SonoSooS** [gbdev/pandocs#525]
In the Power Up Sequence page, at the bottom is the hardware register table.

For example for the HDMA registers, it doesn't make sense that they read as $FF on DMG, as every register that doesn't exist reads as $FF, so it's slightly misleading. Technically correct, but nonsense.

It would be better if these registers would get flagged as `--` or `N/A` or something similar, perhaps with a clarification at the end of the table that all undefined registers read as $FF.

If `N/A` in capital letters is chosen, it's difficult with vision problems to differentiate at a first glance from just a regular hexadecimal value in nearby cells, as it has the same height, and almost the same width, very much blending in with the other nearby cells.
Adding a footnote link (the superscripted tiny number to the right of the cell contents) should be a good step to make it slightly easier to differentiate if no non-intrusive differentation method is available.

**Comment by ISSOtm** [comment]
Sure, `--` should work too. (Perhaps three, so they stand out even if not in a \<code\> block.)

**Comment by calc84maniac** [comment]
I feel like I should bring this up since it's somewhat related, but the CGB-only registers in the CGB / AGB column are also misleading/technically correct, for DMG mode only (which isn't mentioned anywhere). Some registers are marked as ?? with a footnote for compatibility mode, but others are marked as $FF with no indication that it's not the actual value. I think it would be more useful to list the actual values of all these registers on CGB / AGB, and describe the $FF behavior in compatibility mode under the footnote for this issue.

**Comment by ISSOtm** [comment]
I believe the GBC-only regs are still R/W in DMG mode, just inoperative? I'm not certain, though.

**Comment by calc84maniac** [comment]
No, they read as $FF and writes are ignored. I believe the current values were taken directly from the source code of a test ROM which runs in DMG mode, which was a bit of an oversight.
